test/integration/invoker/NodeInvoker.js

```diff
@@ -9,7 +9,7 @@ describe('NodeInvoker Integration', function() {
             let nodeInvoker = new NodeInvoker();
             let testName = 'myTestFn';
             let params = [1, 2];
-            let projectName = path.join(process.cwd(), 'test', 'integration', 'my artifacts', 'testModule');
+            let projectName = path.join(process.cwd(), 'test', 'integration', 'artifacts', 'invoker', 'testModule');
```

Comment:
I'd rather still have an artifact name with spaces somewhere just to test that spaces are handles correctly

Reply:
Added tests to test this particular case.